versions/latest/modules/en/nav.adoc

```diff
@@ -49,6 +49,7 @@
 *** xref:installation-and-upgrade/infrastructure-setup/mysql-database-in-amazon-rds.adoc[]
 *** xref:installation-and-upgrade/infrastructure-setup/amazon-elb-load-balancer.adoc[]
 *** xref:installation-and-upgrade/infrastructure-setup/configure-layer-7-nginx-load-balancer.adoc[]
+*** xref:installation-and-upgrade/infrastructure-setup/configure-oidc-provider.adoc[]
```

Comment:
All the other pages under `installation-and-upgrade/infrastructure-setup` are about setting things up for Rancher itself.  While not a perfect fit since this page is about using Rancher as an authentication service, `rancher-admin/users/authn-and-authz` might be a better choice since it's about authentication (albeit authenticating against Rancher).

Reply:
Good point.